### PR TITLE
docs: clarify safety measures and trust signals for Marketplace extensions

### PR DESCRIPTION
**Pull request by seaniyer**

### CODE REVIEW

**Comment by seaniyer** [comment]
@isidorn  @mariaghiondea  @skoffman1 Please take a look. A couple of tweaks to clarify scan and reflect hardened domain verification

**Review by mariaghiondea — APPROVED** [review]

**Review by isidorn — DISMISSED** [review]

**Comment by isidorn** [comment]
This looks great, thank you @seaniyer 
I am also assigning to @ntrogh for the final review

File: docs/editor/extension-marketplace.md [hunk 1]
```diff
@@ -365,6 +364,9 @@ The Marketplace also provides you with resources to make an informed decision ab
 * **Ratings & Review** - Read what others think about the extension.
 * **Q & A** - Review existing questions and the level of the publisher's responsiveness. You can also engage with the extension's publisher(s) if you have concerns.
 * **Issues, Repository, and License** - Check if the publisher has provided these and if they have the support you expect.
+* **Verified Publisher** - Use the blue check mark next to the publisher name and domain as an additional signal of trust. It indicates that the publisher has proven domain ownership to Marketplace. It also shows that the Marketplace has verified both the existence of the domain and the good standing of the publisher on the Marketplace for at least six months.
```
Comment (ntrogh) [inline docs/editor/extension-marketplace.md:367]:
```suggestion
* **Verified Publisher** - Use the blue check mark next to the publisher name and domain as an additional signal of trust. It indicates that the publisher has proven domain ownership to the Marketplace. It also shows that the Marketplace has verified both the existence of the domain and the good standing of the publisher on the Marketplace for at least six months.
```

**Review by ntrogh — DISMISSED** [review]
Just a minor text edit. Otherwise, looks good to me.

**Review by isidorn — APPROVED** [review]

**Review by ntrogh — APPROVED** [review]

**Pull request merged**